quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/MicroProfileConfigProvider.java

```diff
@@ -20,14 +20,16 @@
 import static org.keycloak.quarkus.runtime.configuration.Configuration.OPTION_PART_SEPARATOR;
 import static org.keycloak.quarkus.runtime.configuration.Configuration.toDashCase;
 
+import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
-import org.eclipse.microprofile.config.ConfigProvider;
-
+import org.eclipse.microprofile.config.ConfigValue;
 import org.keycloak.Config;
 import org.keycloak.Config.Scope;
 
+import io.smallrye.config.SmallRyeConfig;
+
 public class MicroProfileConfigProvider implements Config.ConfigProvider {
```

Comment:
```suggestion
public class SmallRyeConfigProvider implements Config.ConfigProvider {
```

So, shouldn't be it SmallRye config provider? :P

Reply:
> So, shouldn't be it SmallRye config provider? :P

It could be, but I don't think we need to update the class, factory, and other references, as there's no practical difference to us between MP and SmallRye.